index.bs

```diff
@@ -147,11 +145,80 @@ In general, it is recommended to use properties instead of [=Metadata OBUs=] in
+  - The file contains at least one [=AV1 Image Sequence=].
+  - The [=primary image=] is either an [=AV1 Image Item=] or a derived image that references one or more coded images that all are [=AV1 Image Items=].
+  - The AVIF file conforms to at least one AVIF profile, and the profile is listed in the [=compatible_brands=] field of the [=FileTypeBox=].
+  - The AVIF file specifies 'avif' in the [=major_brand=] field of the [=FileTypeBox=].
+
+  <h3 id="non-conforming-files">AVIF files not confirming to a profile</h3>
+AVIF files that do not conform to at least one AVIF profile shall not use the ".avif" or ".avifs" file extensions. Such files may use the ".heif" or ".heifs" file extensions, as defined in Annex D and E of [[!HEIF]].
+
+  <h2 id="profiles-and-brands">Profiles and brands</h2>
+
+  The profiles defined in this section are for enabling interoperability between [=AV1 Image File Format=] files and [=AV1 Image File Format=] readers/parsers.
+  A profile imposes a set of specific restrictions.
+  The presence of a brand indicating a profile can be interpreted as the permission for those [=AV1 Image File Format=] readers/parsers and [=AV1 Image File Format=] renderers that only implement the features required by the profile, to process the corresponding file.
+  An [=AV1 Image File Format=] file may conform to multiple profiles. Similarly, an [=AV1 Image File Format=] reader/parser or [=AV1 Image File Format=] renderer may be capable of processing one or more profiles.
+
+  The following constraints are common to all profiles defined in this specification:
+  - The file shall be compliant with the [[!MIAF]] specification and list 'miaf' in the [=compatible_brands=] field of the [=FileTypeBox=].
+  - The [=primary image=] shall be either an [=AV1 Image Item=] or a derived image that references one or more coded images that all are [=AV1 Image Items=].
+
+  <h3 id="AVIF-baseline-profile">AVIF Baseline Profile</h3>
+
+<p>This section defines the MIAF AV1 Baseline profile of [[!HEIF]], specifically for [[!AV1]] bitstreams, based on the constraints specified in [[!MIAF]] and identified by the brand  <dfn value export for="AV1 Image Item Type">MA1B</dfn>.</p>
+
+<p>If the brand <code>MA1B</code> is in the list of [=compatible_brands=] of a TrackTypeBox or [=FileTypeBox=], the common constraints in the Profiles and brands section shall apply.
+
+  The following additional constraints apply to all [=AV1 Image Items=] and all [=AV1 Image Sequences=]:
+  - The [=AV1 Image Item=] is the content of a single valid [=AV1 Sync Sample=] as defined in [[!AV1-ISOBMFF]].
+  - The [=AV1 Image Item=] shall have only one [=Sequence Header OBU=].
+  - The [=AV1 Image Item=] shall have its <code>[=still_picture=]</code> flag set to 1.
+  - The [=AV1 Image Item=] should have its <code>[=reduced_still_picture_header=]</code> flag set to 1.
+  - The AV1 profile shall be the Main Profile and the level shall be 5.1 level at Main tier or lower.
```

Comment:
As discussed, we should add a NOTE saying that tiers are not constrained because not relevant, given that timing is optional in image sequence and not relevant in image items/collections.